kratos/containers/variables_list_data_value_container.h

```diff
@@ -694,6 +694,11 @@ class KRATOS_API(KRATOS_CORE) VariablesListDataValueContainer
         return mpData;
     }
 
+    const BlockType* Data() const
```

Comment:
You don't want the pointer to be constant?

Reply:
`const BlockType * const`?

Reply:
I am not opposed either way, but is it a problem if the caller reassigns the pointer? I assumed it was irrelevant from the point of view of the `VariablesListDataValueContainer`.

Reply:
You are right! In this case is not important if the return pointer is not constant